src/Mollie/WC/Plugin.php

```diff
@@ -485,7 +511,7 @@ public static function onMollieReturn ()
         // Add utm_nooverride query string
         $redirect_url = add_query_arg(['utm_nooverride' => 1], $redirect_url);
 
-        self::debug(__METHOD__ . ": Redirect url on return order " . $gateway->id . ", order $orderId: $redirect_url");
+        debug(__METHOD__ . ": Redirect url on return order " . $gateway->id . ", order $orderId: $redirect_url");
```

Comment:
I would use interpolation for `$gateway->id` and remember to use curly braces around the variables in a string to avoid possible problems.